Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/model/Settings.java

```diff
@@ -184,6 +259,31 @@ public void saveSettings(SettingsActivityView view, Context context)
 
       NativeConfig.save(NativeConfig.LAYER_LOCAL_GAME);
     }
+
+    saveWiimoteProifleSettings();
+  }
+
+  public void saveWiimoteProifleSettings()
```

Comment:
```suggestion
  public void saveWiimoteProfileSettings()
```

Reply:
Yeah, I might be a little too sleep deprived right now. I'll revisit this later today.